tequilapi/http_middlewares.go

```diff
@@ -40,3 +42,26 @@ func generatePreflightResponse(req *http.Request, resp http.ResponseWriter) {
 		resp.Header().Add("Access-Control-Allow-Headers", headerValue)
 	}
 }
+
+const cacheControlHeader = "Cache-control"
+const noStore = "no-store"
+const noCache = "no-cache"
+const mustRevalidate = "must-revalidate"
+
+type cacheControl struct {
+	originalHandler http.Handler
+}
```

Comment:
This is duplicated with `corsHandler` - only `ServeHTTP` differs.

We can DRY this by having
```go
type handlerWrapper struct {
  originalHandler http.Handler
  serveHttp func(resp http.ResponseWriter, req *http.Request)
}

func DisableCaching(original http.Handler) http.Handler {
  return &handlerWrapper{original, serveDisabledCaching}
}

func serveDisabledCaching...
```
and same applies for `ApplyCors`.

Reply:
Possible. But only two structures at the moment. Maybe when count reaches 3 :)